client/lib/site/index.js

```diff
@@ -218,4 +218,8 @@ Site.prototype.isUpgradeable = function() {
 	return this.capabilities && this.capabilities.manage_options;
 };
 
+Site.prototype.isCustomizable = function() {
+	return !! ( this.capabilities && this.capabilities.edit_theme_options );
+};
```

Comment:
We had to revert the PR that was removing computed attrs. I think we can restore the selector, though.